src/test/java/com/puppycrawl/tools/checkstyle/AstTreeStringPrinterTest.java

```diff
@@ -70,6 +73,20 @@ public void testParseFile() throws Exception {
                 JavaParser.Options.WITHOUT_COMMENTS);
     }
 
+    @Test
+    public void testPrintBranch() throws Exception {
+        final DetailAST ast = JavaParser.parseFile(
+            new File(getPath("InputAstTreeStringPrinterPrintBranch.java")),
+            JavaParser.Options.WITHOUT_COMMENTS);
```

Comment:
Do you mean additional test or just branch with comments?
Because changing the option won't make any difference since test file does not have comments in it.